.vscode/extensions.json

```diff
@@ -2,6 +2,17 @@
   "recommendations": [
     "editorconfig.editorconfig",
     "redhat.vscode-yaml",
-    "biomejs.biome"
+    "biomejs.biome",
+    "github.remotehub",
+    "ms-vscode.remote-repositories",
+    "github.vscode-pull-request-github",
+    "gitlab.gitlab-workflow",
+    "eamodio.gitlens",
+    "mhutchie.git-graph",
+    "streetsidesoftware.code-spell-checker",
+    "bradlc.vscode-tailwindcss",
+    "natqe.reload",
+    "johnpapa.vscode-peacock",
+    "anseki.vscode-color"
```

Comment:
As someone is specifically trying to get people to NOT implement custom colors and use theme vars instead, this is going to need some really strong justification

Reply:
i only use it for visualizing the color; the package you suggested does that as well; happy to go with that one instead